src/Interpreters/Context.cpp

```diff
@@ -6933,3 +6933,15 @@ Poco::Timespan HTTPContext::getSendTimeout() const
 }
 
 }
+
+namespace DB
+{
+namespace Setting
+{
+    extern const SettingsString default_dictionary_database;
+}
+}
```

Comment:
There is already defined namespace for settings at the beginning of the file, the setting should be declared there